### PR TITLE
Support extensions in main process

### LINKED ISSUE #1024: Extensions should be able run on main process

### PR DESCRIPTION
**Pull request by jakolehm**
Fixes #1024

### CODE REVIEW

File: src/extensions/rollup.config.ts [hunk 1]
```diff
@@ -22,6 +22,19 @@ const config: RollupOptions = {
   ],
 };
 
+const rendererConfig: RollupOptions = {
+  input: "src/extensions/extension-renderer-api.ts",
+  output: [
+    { file: "types/extension-renderer-api.d.ts", format: "es", }
```
Comment (ixrock) [inline src/extensions/rollup.config.ts:28]: What the output it produces? Isn't it should be wrapped somehow like `dtsModuleWrap({ name: "@lens/extensions" })`?

**Review by ixrock — APPROVED** [review]
LGTM 👍

File: src/extensions/page-store.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export interface PageComponents {
   MenuIcon: React.ComponentType<IconProps>;
 }
 
-export class PagesStore {
+export class PageStore {
```
Comment (Nokel81) [inline src/extensions/page-store.ts:30]: should this be an `extends Singleton`?

File: src/extensions/rollup.config.ts (hunk 1 above)
Comment (jakolehm) [reply src/extensions/rollup.config.ts:28]: Not sure what you mean, it's using `dtsModuleWrap({ name: "@lens/ui-extensions" })`.

Comment (ixrock) [reply src/extensions/rollup.config.ts:28]: Ok, nevermind, I thought it was inside previous config :)

**Pull request merged**